sqlglot/dialects/snowflake.py

```diff
@@ -502,6 +502,7 @@ class Snowflake(Dialect):
             expr_type: lambda self, e: self._annotate_by_args(e, "this")
             for expr_type in (exp.Reverse,)
         },
+        exp.ConcatWs: lambda self, e: self._annotate_concat_ws(e),
```

Comment:
I believe we can simply do:

```python
exp.ConcatWs: lambda self, e: self._annotate_by_args(e, "expressions")
```

Don't think that we need `_annotate_concat_ws`. Lemme know if I missed something.